src/main/scala/cromwell/engine/backend/jes/Pipeline.scala

```diff
@@ -36,7 +36,8 @@ object Pipeline extends LazyLogging {
                  projectId, 
                  gcsPath, 
                  workflow, 
-                 call, 
+                 call,
+                 key,
```

Comment:
Does this still need the `Call` if it has the `CallKey`?  I remember when I briefly looked at this before it seemed like most things only used the `Call` to get to the FQN.